src/condor_utils/proc_family_direct_cgroup_v2.cpp

```diff
@@ -237,6 +237,8 @@ ProcFamilyDirectCgroupV2::cgroupify_process(const std::string &cgroup_name, pid_
 			dprintf(D_ALWAYS, "Error writing procid %d to %s: %s\n", pid, procs_filename.c_str(), strerror(errno));
 			close(fd);
 			return false;
+		} else {
+			dprintf(D_ALWAYS, "Successfully moved procid %d to cgroup %s\n", pid, procs_filename.c_str());
```

Comment:
D_ALWAYS for a successful operation?  Should probably become something else before this knob becomes the default.